dependencies.gradle

```diff
@@ -9,6 +9,7 @@ ext.versions = [
 
         gradle              : '3.4',
         support             : '25.2.0',
+        constraint          : '1.0.0-beta4',
```

Comment:
Why beta? Constraints is released http://tools.android.com/recent/constraintlayout10isnowavailable